docs/docs/rest-api/public/api/v2/pods.raml

```diff
@@ -0,0 +1,22 @@
+#%RAML 1.0
```

Comment:
Nothing? I really don't like the idea of listing hundreds to thousands of pods in one massive json, don't see why not though...